server/sonar-server/src/main/java/org/sonar/server/computation/task/container/TaskContainerImpl.java

```diff
@@ -32,16 +32,16 @@
 import org.sonar.core.platform.ComponentContainer;
 import org.sonar.core.platform.ContainerPopulator;
 import org.sonar.core.platform.Module;
+import org.sonar.core.platform.StopSafeReflectionLifecycleStrategy;
 
 import static java.util.Objects.requireNonNull;
 
-public class TaskContainerImpl extends ComponentContainer implements TaskContainer {
+public class TaskContainerImpl extends ComponentContainer implements TaskContainer, AutoCloseable {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "AutoCloseable" is already extended by "TaskContainer"; there is no need to implement it here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1939)